src/php/NodeEnvironment.php

```diff
@@ -8,9 +8,9 @@
 
 final class NodeEnvironment
 {
-    public const CTX_EXPR = 'expr';
-    public const CTX_STMT = 'stmt';
-    public const CTX_RET = 'ret';
+    public const CONTEXT_EXPRESSION = 'expr';
```

Comment:
You could also rename the strings to 'expression', 'statement' and 'return'. I don't think the strings are used somewhere other than with these constants.

Reply:
Agreed! :)